api/MediaKeyStatusMap.json

```diff
@@ -109,10 +109,10 @@
               "version_added": "13"
             },
             "firefox": {
-              "version_added": null
+              "version_added": "47"
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1223980#c1 supports `forEach` being special and landing later, so I won't double check exactly when this was added.